docs/external-template-creation.md

```diff
@@ -1,46 +1,35 @@
-# Creating an External Template
+# Creating a new template
 
-`packages/templates/src/raw` is a dumping ground for raw story json output from the editor, which is then interpreted as a template.
+## Where things are stored
 
-## Getting The Raw Story JSON
+- The **story JSON representation** for each template is stored in [`packages/templates/src/raw/`](https://github.com/google/web-stories-wp/tree/main/packages/templates/src/raw) (in the `main` branch).
+- The **SVGs** used in each template are stored in [`assets/src/edit-story/stickers/`](https://github.com/google/web-stories-wp/tree/main/assets/src/edit-story/stickers) (in the `main` branch).
```

Comment:
Are we ditching adding new shapes all together (like our original maskable shapes)? if not, happy to add a section on how to create shapes although it's pretty involved and requires some svg editing software and running a node script to capture the high amount of precision needed to draw complex paths in a 0 -> 1 coordinate space.

Reply:
We aren't. Since there will be some new masks in the templates, it would be beneficial to also mention how to create the shapes. If you could add a section for this, that would be great!

Reply:
Awesome. I'll create a ticket for this and add it to the sprint. it's pretty involved. I'll add a PR too with the little node script we use to normalize the svg path coords with the proper precision

Reply:
https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/7228